celery_utils/management/commands/drop_djcelery_tables.py

```diff
@@ -0,0 +1,80 @@
+"""
+This command will drop the tables included in django-celery's
+migrations, which are included when we bump that to 3.2.1
+
+If the tables are not empty, this command will show a message and exit.
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from collections import OrderedDict
+import logging
+from textwrap import dedent
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+from djcelery.models import (CrontabSchedule, IntervalSchedule, PeriodicTask,  # pylint:disable=import-error
+                             PeriodicTasks, TaskMeta, TaskSetMeta, TaskState, WorkerState)
+
+log = logging.getLogger(__name__)
+
+DJCELERY_MODEL_TABLES = OrderedDict({
+    PeriodicTask: 'djcelery_periodictask',  # must be before crontab, interval
+    TaskState: 'djcelery_taskstate',  # must be before workerstate
+})
+DJCELERY_MODEL_TABLES.update({
+    CrontabSchedule: 'djcelery_crontabschedule',
+    IntervalSchedule: 'djcelery_intervalschedule',
+    PeriodicTasks: 'djcelery_periodictasks',
+    WorkerState: 'djcelery_workerstate',
+    TaskSetMeta: 'celery_tasksetmeta',
+    TaskMeta: 'celery_taskmeta',
+})
+
+
+class Command(BaseCommand):
+    """
+    Drop the database tables used by django-celery, iff they are empty.
+    """
+    help = dedent(__doc__).strip()
+
+    def _log_execute(self, cursor, sql, context=""):
+        """
+        Log a given SQL input (as string) before executing it.
+        """
+        log.info("{} raw SQL:\n{}".format(context, sql))
+        cursor.execute(sql)
+
+    def handle(self, *args, **options):
+        with connection.cursor() as cursor:
+            try:
+                log.info("Acquiring tables locks on {}".format(", ".join(DJCELERY_MODEL_TABLES.values())))
+                lock_sql = "LOCK TABLES {}".format(
+                    ", ".join(
```

Comment:
Nit: I typically would quote the table names in the SQL. Isn't a problem with your current data set but good practice.